content/pl/port-binding.md

```diff
@@ -0,0 +1,14 @@
+## VII. Przydzielanie portów
+### Eksportuj usługi przez przydzielenie portów
+
+Aplikacje internetowe czasem uruchamiane są wewnętrz webserwera.  Aplikacje napisane w PHP np. działają jako moduł wewnątrz [Apache HTTPD](http://httpd.apache.org/), natomiast aplikacje w Javie mogą być uruchomione wewnątrz [Tomcata](http://tomcat.apache.org/).
+
+**Aplikacja dwunastu aspektów jest zupełnie niezależna** i może działać bez wstrzykiwania jej jako zależności w środowisku webserwera. Aplikacja internetowa **eksportuje HTTP jako usługę przez przydzielenie portu** i nasłuchuje zapytań przychodzących do niego zapytań.
+
+W lokalnym środowisku developerskim, programista odwiedza URL taki jak `http://localhost:5000/` by skorzystać z usługi wyeksportowanej przez jego aplikację. Przy wdrożeniu, warstwa nawigacji obsługuje zapytania z wystawionej publicznie nazwy hosta do procesu sieciowego przydzielonego do portu.
+
+Zazwyczaj jest to implementowane za pomocą [deklaracji zależności](./dependencies) dodając bibliotekę webservera do aplikacji, jak np. [Tornado](http://www.tornadoweb.org/) w Pythonie, [Thin](http://code.macournoyer.com/thin/) w Rubym, lub [Jetty](http://jetty.codehaus.org/jetty/) dla Javy i innych języków opierających się na JVM.  Dzieje się to zupełnie w *części użytkownika*, czyli kodzie aplikacji. Kontraktem ze środowiskiem wykonawczym jest przydzielenie portu by obsłużyć zapytanie.
```

Comment:
1. `Zazwyczaj jest to implementowane` - brzmi bardzo kolokwialnie
2. `..implementowane za pomocą deklaracji zależności poprzez dodanie biblioteki webservera do aplikacji...`
3. `Dzieje się to w części użytkownika` - nie mam bladego pojęcia czym jest część użytkownika